creating/creating_index.adoc

```diff
@@ -71,6 +71,34 @@ RUN yum install -y epel-release && \
     yum -y clean all
 ```
 
+There are several package managers beyond yum that should be of note: dnf, rvm, gems, cpan, pip. Most of these 
+managers have some form of a clean up command that will handle excess cache created while performing their package management duties. 
+
+Below are examples pictured for dnf and rvm:
+
+```
+RUN rpm -ivh https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm && \
```

Comment:
Would it make more sense if this example only had dnf in it and not yum to drive your point home?